README.md

```diff
@@ -13,3 +13,13 @@ network. It should have the following properties:
   to enable more efficient communication when possible.
 - **Documented.** The [list-methods](https://localhost:8993/direct) rpc
   method should list out all known methods.
+
+## Installing on different versions
+
+Currently there is support for 2.18 and 2.25+ (this may run on versions between, but has not been tested).
```

Comment:
❓ Should this be 2.19 instead of 2.18?

Reply:
yes this should be 2.19, good catch. I suspect it will work for quite a few versions past 2.19, but i haven't tested them. Primarily was making sure master + 2.18 were good.